setup.py

```diff
@@ -0,0 +1,58 @@
+from setuptools import setup, find_packages
+from codecs import open
+from os import path
+
+here = path.abspath(path.dirname(__file__))
+
+# Get the long description from the README file
+with open(path.join(here, "README.md"), encoding="utf-8") as f:
+    long_description = f.read()
+
+setup(
+    name="ukrainian_word_stress",
+    # Versions should comply with PEP440. For a discussion on single-sourcing
+    # the version across setup.py and the project code, see
+    # https://packaging.python.org/en/latest/single_source_version.html
+    version="0.0.1",
+    description="Find word stress for texts in Ukrainian",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/lang-uk/ukrainian-word-stress",
+    author='Oleksiy Syvokon',
+    author_email='oleksiy.syvokon@gmail.com',
+    license="MIT",
+    # See https://pypi.python.org/pypi?%3Aaction=list_classifiers
+    classifiers=[
+        "Development Status :: 3 - Alpha",
+        "Intended Audience :: Developers",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "License :: OSI Approved :: MIT License",
+    ],
+    keywords="ukrainian nlp word stress accents dictionary linguistics",
+    packages=find_packages(exclude=["docs", "tests"]),
+    package_data={
+        "ukrainian_word_stress": [
+            "data/stress.v2.trie",
+        ]
+    },
+    include_package_data=True,
+    install_requires=[
```

Comment:
It's better to read those from requirements.txt

Also, add extra requirements for the building of the dictionary (TQDM?)

Reply:
Added tqdm to dev requirements. As of reading from requirements.txt -- with just two dependencies, it's an overkill, IMO.